src/main/resources/public/js/pages/organization-search.js

```diff
@@ -103,10 +103,10 @@ class OrganizationSearch extends React.Component {
                             this.setState({organizationSelected: value})
                         }}
                         placeholder={i18n._(t`search.organization.search-organization`)}
-                    />
+                    />{i18n._(t`search.organization.not.found-organization`)}
                     <div className="flex-row options">
                         <Link to="/my/organization/create" className={"new-organization"}>
-                            {i18n._(t`organization.search.new`)}
+                            <button type="submit" className="btn btn-submit">{i18n._(t`search.organization.create-organization`)}</button>
```

Comment:
a submit button has a very specific purpose. here you just want to represent a link as a button. the styling has to be done at the <Link> component level.